src/devices/sound/qsoundhle.h

```diff
@@ -34,23 +35,133 @@ class qsound_hle_device : public device_t, public device_sound_interface, public
 	virtual void rom_bank_updated() override;
 
 private:
+
+	// DSP ROM sample map
+	enum {
```

Comment:
Please follow Allman style and put the opening brace on the following line, as is the case for the device class itself.